### PR TITLE
Breaking: change num_output with quantiles in TFT (mxnet)

### PR DESCRIPTION
**Pull request by baniasbaabe**
*Issue #, if available:*

*Description of changes:*
The TFT model with MXNet implementation, the quantiles are calculated based on `num_outputs`. This is a bit counter-intuitive, so why can I not give the model custom quantiles? Another limitation was that the automatically calculated quantiles could return negative values for high `num_outputs`. So I change the attribute to `quantiles` where you can give the model a list of floats (like MQCNN).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

File: src/gluonts/mx/model/tft/_network.py [hunk 1]
```diff
@@ -151,7 +151,8 @@ def __init__(
         d_var: int,
         d_hidden: int,
         n_head: int,
-        n_output: int,
+        #n_output: int,
```
Comment (lostella) [inline src/gluonts/mx/model/tft/_network.py]: Could this be removed instead of commented?

File: src/gluonts/mx/model/tft/_network.py [hunk 2]
```diff
@@ -167,11 +168,8 @@ def __init__(
         self.d_var = d_var
         self.d_hidden = d_hidden
         self.n_head = n_head
-        self.n_output = n_output
-        self.quantiles = sum(
-            ([i / 10, 1.0 - i / 10] for i in range(1, (n_output + 1) // 2)),
-            [0.5],
-        )
+        # self.n_output = n_output
```
Comment (lostella) [inline src/gluonts/mx/model/tft/_network.py]: Remove?

**Review by lostella — CHANGES_REQUESTED** [review]
Hi @baniasbaabe, thanks for the PR! I generally agree with this change, I only have a couple of comments inline.

Note that this would be a breaking change: if merged, it will not be included in a release before 0.14.0.

**Comment by lostella** [comment]
@baniasbaabe just out of curiosity, is there any particular reason to prefer the MXNet implementation of TFT over the [PyTorch one](https://github.com/awslabs/gluonts/tree/dev/src/gluonts/torch/model/tft)?

File: src/gluonts/mx/model/tft/_network.py (hunk 1 above)
Comment (baniasbaabe) [reply src/gluonts/mx/model/tft/_network.py]: Yes, it sneaked into it :)

File: src/gluonts/mx/model/tft/_network.py (hunk 2 above)
Comment (baniasbaabe) [reply src/gluonts/mx/model/tft/_network.py]: Yes

**Comment by baniasbaabe** [comment]
> @baniasbaabe just out of curiosity, is there any particular reason to prefer the MXNet implementation of TFT over the [PyTorch one](https://github.com/awslabs/gluonts/tree/dev/src/gluonts/torch/model/tft)?

I'm not a big fan of MXNet, but there are indeed some organizations that rely on MXNet and maybe build their model pipelines around it. Most of the gluonts models were MXNet based some versions ago :)

**Review by lostella — DISMISSED** [review]

**Review by lostella — APPROVED** [review]
Thanks @baniasbaabe, and sorry it took a bit long but this can be merged 🚀  I just added minor styling changes

**Pull request merged**